src/components/dashboard/PotentialBarriers.vue

```diff
@@ -19,20 +18,22 @@
       </ul>
     </div>
 
-    <table class="mt-4 centered table is-narrow">
-      <tbody>
-        <tr>
-          <th>Potential Barrier</th>
-          <th>Community</th>
-          <th>State</th>
-        </tr>
-        <tr v-for="(row, i) in rows" :key="i">
-          <td>{{ row.fieldName }}:</td>
-          <td class="pl-2">{{ formatPct(barrier[row.property]) }}</td>
-          <td class="pl-2">State number here</td>
-        </tr>
-      </tbody>
-    </table>
+    <div class="table-container m-auto">
+      <table class="table is-narrow">
+        <tbody>
+          <tr>
+            <th>Potential Barrier</th>
+            <th>Community</th>
+            <th>State</th>
+          </tr>
+          <tr v-for="(row, i) in rows" :key="i">
+            <td>{{ row.fieldName }}:</td>
+            <td class="pl-2">{{ formatPct(barrier[row.property]) }}</td>
+            <td class="pl-2">State number here</td>
```

Comment:
it looks like the values are top aligned, maybe make middle aligned vertically

Reply:
i set vertical alignment to middle !